### ISSUE TITLE
Redirect Request: CLP secure signin

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#16494]
### Status
**[2024-07-19] [Fran]** Unblocked per Danielle T:  We are publishing the new CLP tomorrow morning and then you can go ahead and implement the 2nd redirect of https://www.va.gov/initiatives/sign-in-securely-with-logingov/ to https://va.gov/initiatives/prepare-for-vas-secure-sign-in-changes/.
**[2024-04-09]** On pause per Danielle; she shared it's part of a big part of the communications plan and she will tell us as soon as they have the go-ahead.

### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

### Type of request
- [x] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [x] We need a custom vanity URL (complete vanity URL section)

### Implementation date
When does this request need to be live?
Choose one:
- [x] On a specific date, connected to a program or publishing requirement. **DATE:** **TBD**, CAIA will give us the greenlight to confirm the landing page we're redirecting to is published and we can deploy / merge
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/initiatives/sign-in-securely-with-logingov/  | https://va.gov/initiatives/prepare-for-vas-secure-sign-in-changes/ (edited JM 12/19)

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
https://va.gov/sign-in-changes/ (edited JM 12/19) |  https://va.gov/initiatives/prepare-for-vas-secure-sign-in-changes/ (edited JM 12/19)

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [ ] IA: Request is vetted and documented and implementation plan is clear 
- [ ] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by jilladams** [comment]
@FranECross new ticket that arose out of #16271 that should be light in refining / points, but could be prob queued for upcoming sprints as stretch work.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @jilladams and team! We're going to need to include the vanity url for this in communications that will go out in early January. We'll need to coordinate the publishing of the page/vanity url with those communications. Let me know your thoughts on how best to work with you all on coordinating timing for that.

**Comment by jennymayoco** [comment]
I updated the URLs with a forward slash at the end. Other than that, URLs look good! @jilladams

**Comment by jilladams** [comment]
@DanielleThierryUSDSVA Noted - we'll have a sprint starting on Jan 3. We can plan this ticket into that sprint, prep a PR and standby for greenlight to merge / deploy it, whenever you all are ready. (If you think your launch date will be Jan 3/4/5, early warning is helpful just to make sure we pick it up early in sprint.)

**Comment by randimays** [comment]
PR is prepped and in draft, pending CAIA's greenlight: https://github.com/department-of-veterans-affairs/devops/pull/13827

**Comment by jilladams** [comment]
@DanielleThierryUSDSVA noting: for secure sign in CLP, we're ready to merge / ship the redirect when you all are ready to publish / redirect the existing CLP.

**Comment by jilladams** [comment]
@strelichl We are just waiting on the greenlight from CAIA to publish the redirect. Are you all ready for that?

**Comment by coforma-terry** [comment]
Hey @jennymayoco - are we able to publish the redirect for this yet? Thanks! :)

**Comment by DanielleThierryUSDSVA** [comment]
Hi all! @jilladams Not quite yet. We're waiting for approval on a press release from OPIA before we can finalize the date for launching the communications that will include this CLP. I'll make sure to tell you as soon as I have a date!

**Comment by davidconlon** [comment]
@jilladams @FranECross I believe that whenever this drops this will need to be pushed out as a redirect that day as opposed to waiting for the weekly redirect process (if that's still a thing?) can you elaborate here if I am misunderstanding the redirect platform process.

Ultimately we want to be sure that the redirects go out quickly (within the bounds of a normal daily deploy, not OOB)

**Comment by randimays** [comment]
@davidconlon The deploy schedule for same-domain redirects (va.gov -> va.gov) is Wednesday mornings (10am CST I believe). Any other day/time is OOB.

**Comment by davidconlon** [comment]
@randimays change to this (sorry for 11th hour change) 

Right now this issue is redirecting two urls:
Vanity URL: https://va.gov/sign-in-changes/  to https://va.gov/initiatives/prepare-for-vas-secure-sign-in-changes/
OLD CLP URL redirect: https://www.va.gov/initiatives/sign-in-securely-with-logingov/ to https://va.gov/initiatives/prepare-for-vas-secure-sign-in-changes/

We want to separate these two so that 1 (Vanity URL) ships Today and goes live (1/12) and 2 (OLD CLP Redirect) can ship at a later date TBD.

That way @DanielleThierryUSDSVA can do a content publish of the new page whenever OPIA is ready, and we can redirect the OLD CLP when that's done as part of a Weds release

**Comment by randimays** [comment]
Vanity URL: https://va.gov/sign-in-changes/ to https://va.gov/initiatives/prepare-for-vas-secure-sign-in-changes/

The vanity URL redirect has been merged separately ([PR](https://github.com/department-of-veterans-affairs/devops/pull/13869)) and is deployed to production.

---------

The OLD CLP URL redirect (below) is awaiting CAIA's green light.

https://www.va.gov/initiatives/sign-in-securely-with-logingov/ to https://va.gov/initiatives/prepare-for-vas-secure-sign-in-changes/

**Comment by randimays** [comment]
End of sprint update: CLP URL redirect (above) is awaiting CAIA's green light and will likely carry over.

**Comment by randimays** [comment]
@DanielleThierryUSDSVA To confirm: we are waiting on OPIA's approval to move forward here?

**Comment by jilladams** [comment]
Removing from sprints until CAIA greenlights merge of final PR here.

**Comment by FranECross** [comment]
@DanielleThierryUSDSVA Just checking back on the status of this request. Thanks in advance!

**Comment by randimays** [comment]
@DanielleThierryUSDSVA Do you have an update on when these redirects can be released?

**Comment by DanielleThierryUSDSVA** [comment]
Hi @randimays. Nope, we're still on pause. Don't worry, though, this is a big part of the communications plan and I will absolutely tell you all as soon as we have the go-ahead!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @randimays @FranECross @jilladams. The day has finally arrived. We are publishing the new CLP tomorrow morning and then you can go ahead and implement the 2nd redirect of https://www.va.gov/initiatives/sign-in-securely-with-logingov/ to https://va.gov/initiatives/prepare-for-vas-secure-sign-in-changes/.

**Comment by jilladams** [comment]
Moved this ticket to the top of ready. We plan a new sprint next Wed (7/24) , so can pull this into that sprint if someone doesn't have capacity to pick it up before then. cc @FranECross

**Comment by DanielleThierryUSDSVA** [comment]
Thank you, @jilladams! Yes, as quickly as we can get this done would be much appreciated. Thanks again!

**Comment by randimays** [comment]
@jennymayoco @DanielleThierryUSDSVA These redirects are in place and working correctly in production for me. Could you verify on your end and close the ticket if all is well?
